docs/src/components/accounts_and_libraries.md

```diff
@@ -0,0 +1,23 @@
+# Accounts and Libraries
+
+- **Valence Account:** An escrow contract that can hold balances for various supported token-types (e.g., in Cosmos `ics-20` or `cw-20`) . Accounts register valence libraries that can perform token operations on the account.
+
+- **Valence Libraries:** Valence libraries are registered with Valence accounts. Accounts are either registered as input accounts or output accounts. Libraries perform token operations on the input accounts, such as splitting, routing, or providing liquidity on a DEX, and depending on the business logic in the Library, they route the resulting tokens to the output accounts. Libraries expose _Functions_ that can be performed on them.
```

Comment:
I think we need to refrain from using input and output accounts, especially in defining libraries.

"Libraries executes a function on accounts"

The fact this function usually or might take funds from account A and move them to account B, doesn't mean its what define libraries.

Good example is the rebalancer library, it has no "output account", it executes a "register" function on an account and that's it.  

Just like programming libraries, you approve them by importing them, which grants them the ability to execute functions where you imported them.

Reply:
Agree that once we move to singleton libraries we probably need to rephrase this to better represent how services / accounts are linked.